packages/ui-components/animate/README.md

```diff
@@ -0,0 +1,19 @@
+# @k8slens/animate
+
+This package contains stuff related to creating Lens-applications. 
+
+# Usage
+
+```bash
+$ npm install @k8slens/animate
+```
+
+```typescript
+export type { AnimateName, AnimateProps } from "@k8slens/button";
+export type { Animate } from "@k8slens/button";
+export { defaultEnterDurationForAnimatedInjectable } from "@k8slens/button";
+export { defaultLeaveDurationForAnimatedInjectable } from "@k8slens/button";
+export { requestAnimationFrameInjectable } from "@k8slens/button";
```

Comment:
`@k8slens/button` -> `@k8slens/animate`

Reply:
Also prefer exporting `injectionToken` and not `injectable` from the feature.

Reply:
@jansav We haven't made a full decision on preferring `injectionTokens` over `injectables`, plus it adds more work that isn't required ATM

Reply:
Exporting `injectable` introduces so many blindspots that we just shouldn't do it. It has been the instruction from the beginning. It's even listed in the [README.md](https://www.npmjs.com/package/@k8slens/feature-core)

Reply:
@Iku-turso Doesn't completely agree as I understand. And both of those of those blindspots could easily be blocked in code instead, which would be even better.